tests/unit/packages/Teams/js/package.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "@microsoft/bot-components-teams-tests",
+  "private": true,
+  "scripts": {
+    "test": "mocha --require ts-node/register tests/*.test.ts"
+  },
+  "dependencies": {
```

Comment:
I felt that since this package is only tests, it made sense to put this all under `dependencies` vs. `devDependencies`, since all of these packages are technically required at runtime. Open to moving.

Reply:
Yep - makes sense.